sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/UnnecessaryUsings.cs

```diff
@@ -151,6 +151,36 @@ public void DoLinqQuery(List<string> myList)
     }
 }
 
+namespace LinqQuery2
+{
+    using global::System.Linq; // Compliant - statement is needed for query syntax
+    class LinqQueryUser
+    {
+        public void DoLinqQuery(List<string> myList)
+        {
+            var linkQuery = from item in myList select item.GetType();
```

Comment:
Little typo here and on other places.
```suggestion
            var linqQuery = from item in myList select item.GetType();
```
Since Q in LINQ stands for Query, I would suggest just `var query = ...`

Reply:
Update name to `query` and `DoQuery` for most. However, I left the namespace names as they are, in order to quickly understand what theses tests are about, even if the name is slightly redundant